Dockerfile_build_ubuntu64_go1.12

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    python-software-properties \
+    software-properties-common \
+    wget \
+    git \
+    mercurial \
+    make \
+    ruby \
+    ruby-dev \
+    autoconf \
+    libtool \
+    build-essential \
+    rpm \
+    zip \
+    python \
+    python-boto \
+    asciidoc \
+    xmlto \
+    docbook-xsl
+
+RUN gem install fpm
+
+# setup environment
+ENV GO_VERSION  1.12
```

Comment:
Does this need to be updated?

Reply:
Actually, the name of this file is `Dockerfile_build_ubuntu64_go1.12 ` so I'm not sure if it should be changed here or a new file created??